src/include/daos_srv/vos.h

```diff
@@ -294,7 +294,7 @@ vos_pool_open(const char *path, uuid_t uuid, unsigned int flags,
  */
 int
 vos_pool_open_metrics(const char *path, uuid_t uuid, unsigned int flags, void *metrics,
-		      daos_handle_t *poh);
```

Comment:
This reads like we are opening the pool metrics.

I would suggest we simply add an entry to pool_open_flags

VOS_POF_SKIP_UUID_CHECK

Reply:
Ya, the name of this function kind of confused me at first. I'll change to use a flag instead.